ui/intermittent-failures/MainView.jsx

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import { Row, Col } from 'reactstrap';
 import PropTypes from 'prop-types';
 import moment from 'moment';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css';
```

Comment:
Is the CSS import needed in all of {MainView.jsx, Layout.jsx, BugDetailsView.jsx}?
It seems like it could just be imported once, in an appropriate location?

Reply:
Good point. I'll move it up to the index level where other CSS is being imported.